debugging/book-library/script.js

```diff
@@ -1,45 +1,34 @@
 let myLibrary = [];
```

Comment:
This is an example of a variable that should be declared as a const. 
There are other variables within script.js that are declared as let but should be const.  
➡️ Can you find them and change them? 
_Using const prevents accidental reassignment of the variable and makes your code easier to read._